cmd/flux/cmd/test.go

```diff
@@ -547,11 +548,13 @@ func (t *TestRunner) Run(executor TestExecutor, verbosity int, skipTestCases []s
 func (t *TestRunner) Finish() {
 	t.reporter.Summarize(t.tests)
 	for _, test := range t.tests {
-		if test.Error() != nil {
-			os.Exit(1)
+		if err := test.Error(); err != nil {
+			// TODO(onelson): would prefer to return the error but don't want to
```

Comment:
I was wrong about this. This function is safe to change. Please do that instead unless I've missed something.

Reply:
I sort of took the fact it was capitalized to say someone _could be calling it_. I guess all it would mean to them is now they are ignoring an error, right? Not a big breaking change per se.